### PR TITLE
New Icon: Gsmarena

### LINKED ISSUE #8224: New Icon: GSMArena
### Brand Name

GSMArena

### Website

https://gsmarena.com

### Popularity Metric

Similarweb rank: **[762](https://www.similarweb.com/website/gsmarena.com/#overview)**

### Official Resources for Icon and Color

None

### Additional Comments

_Thanks 🙏_

### Contributing

No

### PR DESCRIPTION
**Pull request by ripzerotwo**
**Issue:** closes #8224 

**Similarweb rank:** 762 (https://www.similarweb.com/website/gsmarena.com/#overview)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Svg taken from html.
Preview...
gsmarena

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4852,6 +4852,11 @@
                 ]
             }
         },
+        {
+            "title": "Gsmarena",
```
Comment (adamrusted) [inline _data/simple-icons.json:4856]:
```suggestion
            "title": "GSMArena",
```

File: _data/simple-icons.json [hunk 2]
```diff
@@ -4852,6 +4852,11 @@
...
+            "hex": "F3F3F3",
+            "source": "https://www.gsmarena.com/"
```
Comment (adamrusted) [inline _data/simple-icons.json:4858]:
```suggestion
            "source": "https://www.gsmarena.com"
```

File: _data/simple-icons.json [hunk 3]
```diff
@@ -4852,6 +4852,11 @@
                 ]
             }
         },
+        {
+            "title": "Gsmarena",
+            "hex": "F3F3F3",
```
Comment (adamrusted) [inline _data/simple-icons.json:4857]:
```suggestion
            "hex": "D50000",
```
This suggestion is open to interpretation. The orange seems to be their hover state on the logo and buttons.

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 2 above)
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:4858]: The website itself is consistently referred to as "GSMArena.com"; it seems to be only their YouTube channel that's referred to as "GSMArena".
```suggestion
            "source": "https://www.gsmarena.com/",
            "aliases": {
                "aka": [
                    "GSMArena.com"
                ]
            }
```

File: icons/gsmarena.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Gsmarena</title><path d="M20.324 22.992c-.905-.454-12.625-5.27-12.625-5.27a1.275 1.275 0 0 0-.389-.122c-.39-.056-.78.091-1.061.444 0 0-2.672 4.354-3.066 4.948C2.782 23.58 3.2 24 3.726 24h16.38c.644 0 .898-.67.218-1.008ZM19.688 0h-7.743c-.868 0-1.49.28-2.042 1.043L4.05 10.497c-.333.457-.14.985.336 1.185.974.412 2.766.977 3.68 1.388.718.326 1.157.204 1.552-.382l4.092-6.507.49-.005v7.405c0 .924.37 1.279.946 1.54.577.246 4.144 1.773 4.689 1.973.644.246 1.143-.05 1.143-.731V1.289c0-.706-.585-1.289-1.29-1.289Z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/gsmarena.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>GSMArena</title><path d="M20.324 22.992c-.905-.454-12.625-5.27-12.625-5.27a1.275 1.275 0 0 0-.389-.122c-.39-.056-.78.091-1.061.444 0 0-2.672 4.354-3.066 4.948C2.782 23.58 3.2 24 3.726 24h16.38c.644 0 .898-.67.218-1.008ZM19.688 0h-7.743c-.868 0-1.49.28-2.042 1.043L4.05 10.497c-.333.457-.14.985.336 1.185.974.412 2.766.977 3.68 1.388.718.326 1.157.204 1.552-.382l4.092-6.507.49-.005v7.405c0 .924.37 1.279.946 1.54.577.246 4.144 1.773 4.689 1.973.644.246 1.143-.05 1.143-.731V1.289c0-.706-.585-1.289-1.29-1.289Z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 2 above)
Comment (adamrusted) [reply _data/simple-icons.json:4858]: We might get away with just calling the icon GSMArena.com then. Won't need the alias if the full name is part of the main title!

File: _data/simple-icons.json (hunk 1 above)
Comment (PeterShaggyNoble) [reply _data/simple-icons.json:4856]:
```suggestion
            "title": "GSMArena.com",
```

File: _data/simple-icons.json (hunk 2 above)
Comment (PeterShaggyNoble) [reply _data/simple-icons.json:4858]: Agreed; that would be better 👍🏻

File: icons/gsmarena.svg (hunk 4 above)
Comment (PeterShaggyNoble) [reply icons/gsmarena.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>GSMArena.com</title><path d="M20.324 22.992c-.905-.454-12.625-5.27-12.625-5.27a1.275 1.275 0 0 0-.389-.122c-.39-.056-.78.091-1.061.444 0 0-2.672 4.354-3.066 4.948C2.782 23.58 3.2 24 3.726 24h16.38c.644 0 .898-.67.218-1.008ZM19.688 0h-7.743c-.868 0-1.49.28-2.042 1.043L4.05 10.497c-.333.457-.14.985.336 1.185.974.412 2.766.977 3.68 1.388.718.326 1.157.204 1.552-.382l4.092-6.507.49-.005v7.405c0 .924.37 1.279.946 1.54.577.246 4.144 1.773 4.689 1.973.644.246 1.143-.05 1.143-.731V1.289c0-.706-.585-1.289-1.29-1.289Z"/></svg>
```

**Pull request merged**